packages/core/src/storybook/args/index.ts

```diff
@@ -15,6 +15,21 @@ export const colors = ['', ...paletteFamilies, 'NOTVALID']
 
 export const borderRadii = ['', ...borderRadiusValues, 'NOTVALID']
 
+export const borderStyles = [
+  'none',
+  'hidden',
+  'dotted',
+  'dashed',
+  'solid',
+  'double',
+  'groove',
+  'ridge',
+  'inset',
+  'outset',
+  'initial',
+  'inherit',
+]
```

Comment:
Sorry for seeing the PR late. But we should enforce `as const` for every true constant. Provides excellent TypeScript support in IDE while looking at the plausible values for `borderStyles`, even in JS-only projects. IMHO reduces the need to look at the documentation
CC @craigpalermo @trenaryja